src/apns_connection.erl

```diff
@@ -315,7 +315,9 @@ handle_info(Request, State) ->
 
 %% @hidden
 -spec terminate(term(), state()) -> ok.
-terminate(_Reason, _State) -> ok.
+terminate(_Reason, State=#state{queue = QID}) ->
```

Comment:
@sugus-seu: @gadgetci fails because we're compiling with `-Werrors` and `State` is an unused variable.
Please, either call it `_State` or remove the variable assignment altogether.